examples/accounts/accounts.pact

```diff
@@ -20,7 +20,7 @@
      ccy:string
      rowguard:guard
      date:time
-     data
```

Comment:
Was there a reason we had to change this from a string to object?

Reply:
haha: look further. it's `string` in some functions, objects in others. ie a dumpster fire